doc/crud.md

```diff
@@ -31,6 +31,18 @@ CDTDatastoreManager *manager =
 [[CDTDatastoreManager alloc] initWithDirectory:path
                                          error:&error];
 ```
+```swift
+do {
```

Comment:
I think that it would be good to include the same comment `// Create a CDTDatastoreManager using application internal storage path` as is in the Objective C example so that the examples are more consistent with each other.